rudder-web/src/main/scala/bootstrap/liftweb/checks/CheckDIT.scala

```diff
@@ -141,6 +143,37 @@ class CheckDIT(
       case Full(_) => //ok
     }
 
+    //check Root Policy Server entries (from init-policy-server.ldif)
+    val dns = List(
+        "nodeId=root,ou=Nodes,cn=rudder-configuration"
```

Comment:
Well, the config file is lying, because rudder-init does care at all of these properties.